presto-main/src/main/java/com/facebook/presto/execution/SqlQueryManager.java

```diff
@@ -120,10 +124,14 @@
 
     private final SessionPropertyManager sessionPropertyManager;
 
+    private final InternalNodeManager internalNodeManager;
+
     private final Map<Class<? extends Statement>, QueryExecutionFactory<?>> executionFactories;
 
     private final SqlQueryManagerStats stats = new SqlQueryManagerStats();
 
+    private final AtomicBoolean acceptQueries = new AtomicBoolean(false);
```

Comment:
There is a mixed use of the parameterized constructor in presto-main and presto-hive. I guess specifying false is to emphasize on the initial value. I won't mind removing it though.